Mathlib/Combinatorics/SimpleGraph/DeleteEdges.lean

```diff
@@ -82,6 +89,98 @@ theorem edgeFinset_deleteEdges [DecidableEq V] [Fintype G.edgeSet] (s : Finset (
 
 end DeleteEdges
 
+section DeleteIncidenceSet
+
+/-- Given a vertex `x`, remove the edges incident to `x` from the edge set. -/
+abbrev deleteIncidenceSet (G : SimpleGraph V) (x : V) : SimpleGraph V :=
```

Comment:
Let's make this a def rather than an abbrev: I'm conscious many things around here are abbrevs but we're trying to move away from that!